src/application/ApplicationThunks.ts

```diff
@@ -377,6 +379,7 @@ export const loadApplicationConfigThunk = () => async (dispatch: any, getState:
     }
     const state = getState();
     dispatch(setSSOEnabled(config.ssoEnabled, state.application.cachedSSODiscoveryUrl));
+    dispatch(setSSOProviders(config.ssoProviders));
```

Comment:
should we call this even if the list is `[]`?
Wouldn't that make this break for users that do not want to manually specify their providers?